app-store-connect/src/main.rs

```diff
@@ -0,0 +1,469 @@
+use anyhow::Result;
+use app_store_connect::bundle_api::{BundleId, BundleIdPlatform};
+use app_store_connect::certs_api::{self, Certificate, CertificateType};
+use app_store_connect::device_api::Device;
+use app_store_connect::profile_api::{Profile, ProfileType};
+use app_store_connect::{AppStoreConnectClient, UnifiedApiKey};
+use clap::{Parser, Subcommand};
+use std::io::Write;
+use std::path::PathBuf;
+
+#[derive(Parser)]
+#[clap(author, version, about, long_about = None)]
+struct Args {
+    #[clap(subcommand)]
+    command: Commands,
+}
+
+fn main() -> Result<()> {
+    env_logger::init();
+    let args = Args::parse();
+    args.command.run()
+}
+
+#[derive(Subcommand)]
+enum Commands {
+    /// Generates a PEM encoded RSA2048 signing key
+    GenerateKey {
```

Comment:
The naming here is a bit confusing because it is inconsistent. Is this generating a private key for an API Key? The private key for a code signing certificate? The command name should encode the purpose of the private key, as _key_ is too generic.

Reply:
`GenerateKeyAndCertificate`? The purpose of this command is to generate a key, use the certificate api to get a signing certificate and put the private key and certificate into a pem file.

Reply:
Or `GenerateCodeSigningCertificate` or `GenerateSigningCertificate`.